### ISSUE TITLE
blocking event when soft reset is fired

### ISSUE THREAD

**Issue by plainOldCode** [Knowre-Dev/swint-fork#7]
```
case 'softReset':
    if(!that.resetPostRun){
        that.resetPost(function(err) {
            if (err) {
                print(4, err);
                process.exit(-1);
                return;
            }
            process.emit('controlEnd');
        }); 
    }

    break;
```

```
_.resetPost = function(callback) {
    var that = this,
        workers = [];

    this.resetPostRun = true;

    for (var id in cluster.workers) {
        var worker = cluster.workers[id];

        workers.push(worker);
    }

    async.series(
        workers.map(function(v) {
            return function(cb) {
                that.evt.once('complete', function() {
                    cb(null, true);
                });
                v.kill();
            };
        }),
        function(err) {
            callback(err, true);
            that.resetPostRun = false;
        }
    );
};
```

**Comment by plainOldCode** [comment]
for example. Above code is blocking multiple call of resetPost.

**Comment by Kivol** [comment]
Fixed in v1.2.1.
